pkg/operator/ceph/cluster/osd/envs.go

```diff
@@ -151,6 +160,14 @@ func crushDeviceClassEnvVar(crushDeviceClass string) v1.EnvVar {
 	return v1.EnvVar{Name: CrushDeviceClassVarName, Value: crushDeviceClass}
 }
 
+func encryptedDeviceEnvVar(encryptedDevice bool) v1.EnvVar {
+	return v1.EnvVar{Name: EncryptedDeviceEnvVarName, Value: strconv.FormatBool(encryptedDevice)}
+}
+
+func cephVolumeRawEncryptedEnvVar(secretValue string) v1.EnvVar {
+	return v1.EnvVar{Name: CephVolumeEncryptedKeyEnvVarName, Value: secretValue}
```

Comment:
We don't want the secret value to be in plain text on the pod spec. We would need to reference the created k8s secret instead of using a plain text env var.

Reply:
At this point, this is not a Secret yet so we cannot do this...

Reply:
Solved as discussed using the pvc name.